server/src/main/java/org/apache/kafka/server/config/KRaftConfigs.java

```diff
@@ -114,6 +114,10 @@ public class KRaftConfigs {
     public static final String SERVER_MAX_STARTUP_TIME_MS_DOC = "The maximum number of milliseconds we will wait for the server to come up. " +
             "By default there is no limit. This should be used for testing only.";
 
+    public static final String MIN_SLOW_EVENT_TIME_MS_CONFIG = "controller.slow.event.min.ms";
+    public static final int MIN_SLOW_EVENT_TIME_MS_DEFAULT = 200;
```

Comment:
I think we always log when over this threshold. So putting it at 2 seconds or something seems right? ...